### PR TITLE
Ccarfi ethdenver blog

### PR DESCRIPTION
**Pull request by ccarfi**

### CODE REVIEW

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
Hum there is something fishy with the youtube video? 
Here is what tge draft loads: https://621922803cde5d006d10d344--unlock-protocol-com.netlify.app/blog/ethdenver2022

File: unlock-protocol-com/blog/ethdenver2022.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+> “Build finance around communities, not speculators” 
+
+> “Human Rights Fail-Safes-- Like Human Rights Watch with cryptoeconomic teeth”
+
+> “What policies would a decent world run on blockchains need?”
+
+If you are at all interested in governance and the future of decentralized community building, please take a few minutes and [watch Nathan’s presentation.](https://www.youtube.com/watch?v=ROnYDEwnY04&list=PLvTrX8LNPbPlryx_S85pZPfH9yD7qcPsg&index=7)
+
+<iframe style="position: absolute; top: 0; left: 0; bottom: 0; right: 0; width: 100%; height: 100%;" src="https://www.youtube.com/embed/ROnYDEwnY04" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (julien51) [inline unlock-protocol-com/blog/ethdenver2022.md:36]:
```suggestion
<iframe src="https://www.youtube.com/embed/ROnYDEwnY04" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: unlock-protocol-com/blog/ethdenver2022.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+One slide that resonated explored “Ideologies of the Future: Corporate Libertarianism, Synthetic Technocracy & Digital Democracy.” He then spoke of the plurality of funding in relation to communities in the web3 ecosystem and he [published a deeper blog post](https://www.radicalxchange.org/media/blog/political-ideologies-for-the-21st-century/) about this very topic the day before his talk. Take a moment to watch his presentation because there are some profound theories and concepts we should explore as we build technology and web3 communities.
+
+<iframe style="position: absolute; top: 0; left: 0; bottom: 0; right: 0; width: 100%; height: 100%;" src="https://www.youtube.com/embed/MsMsL5v2-Ls" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (julien51) [inline unlock-protocol-com/blog/ethdenver2022.md:45]:
```suggestion
<iframe src="https://www.youtube.com/embed/MsMsL5v2-Ls" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: unlock-protocol-com/blog/ethdenver2022.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+> “Massively scalable systems of human cooperation (unprecedented levels of scale)
+
+> That can achieve directed outcomes.
+
+> While preserving freedom, autonomy and self-direction effort.”
+
+Her talk is relevant for managers and founders working with decentralized ecosystems, such as DAOs, and trying to build or adapt older systems of work organizational structures that will support the unique characteristics of the web3 community.
+
+<iframe style="position: absolute; top: 0; left: 0; bottom: 0; right: 0; width: 100%; height: 100%;" src="https://www.youtube.com/embed/Web77iQLbfs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (julien51) [inline unlock-protocol-com/blog/ethdenver2022.md:64]:
```suggestion
<iframe src="https://www.youtube.com/embed/Web77iQLbfs" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: unlock-protocol-com/public/images/blog/ethdenver2022/temp.md [hunk 4]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (julien51) [inline unlock-protocol-com/public/images/blog/ethdenver2022/temp.md:1]: ??

**Comment by ccarfi** [comment]
i bet i fat-fingered a tag on the video iframe. will check on fix. 

the temp tile is my back to set up that subdirectory in the images folder. will delete.

**Pull request merged**